main.go

```diff
@@ -87,47 +68,76 @@ func parseJSON(s string) {
 			reverseValue = string(val) + reverseValue
 		}
 
-		fmt.Println(key, reverseValue)
+		infoLogger.Println(key, reverseValue)
 	}
+	return nil
+}
 
+/* 	Open the logfile and configure the loggers that will be used
+*
+*	The main difference between types of loggers is the label (eg INFO) and additional data provided .
+ */
+func initLogging() (*os.File, error) {
+	// Open the file as read only
+	file, err := os.OpenFile(logfile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		return nil, err
+	}
+
+	// Format the timestamp to match the UTC format from the waagent log files
+	// Sample out: INFO 2020-07-29 01:10:53.960425Z GATestExt version: 1.0.0.0 main.go:22: Hello World!
```

Comment:
Either put back the version or modify the sample comment above

Reply:
Time Level=Info/Error/etc Version ShortFile Message

Reply:
Make this a variable - `log.LUTC|log.Ldate|log.Ltime|log.Lmicroseconds|log.Lshortfile`